src/Client/ClientBase.cpp

```diff
@@ -2613,7 +2641,12 @@ void ClientBase::init(int argc, char ** argv)
         ("server_logs_file", po::value<std::string>(), "put server logs into specified file")
 
         ("multiline,m", "multiline")
-        ("multiquery,n", "multiquery")
+        // multiquery is used in 2 cases:
+        // old one (for back compatibility): --multiquery --query <SQL>. --multiquery is used as a flag indicating that <SQL>
```

Comment:
The comment should not mention the existing syntax as "old" or "deprecated", it should rather introduce the new syntax as "alternative" or "convenience" over the existing syntax.

But maybe even leave the comment out, it only replicates the docstring in l. 2649.

Reply:
done: removed comment from here